kent2cky/GeoSearch
# #3 GeoSearch

**What does the PR do?**

Consume relevant API endpoints using fetch api.

**Description of Tasks to be completed**

Have the page fetch relevant data from different API endpoints. 

**How should this be manually tested?**

- After cloning the repo, cd into it and run npm install
- After that, run the command gulp and it should be able to open up age page on your web browser.
- Your internet connection should be on.
- Open up the console tab in your browser developer tools.
- There should be some data contained in an array structure displayed there.
- Those are the results of the fetch.

**Any background context you want to provide?**

The fetched data are not yet wired up to the interface so you can only view
them in the console tab of the browser developer tools.

**What are the relevant pivotal tracker stories?**

#3
